src/docfx/lib/HtmlUtility.cs

```diff
@@ -177,7 +179,7 @@ public static HtmlNode ExtractTitle(HtmlNode node)
                     continue;
                 }
 
-                if (child.NodeType == HtmlNodeType.Element && child.Name == "h1")
+                if (child.NodeType == HtmlNodeType.Element && s_titleElementNames.Contains(child.Name))
```

Comment:
nit: `(name == "h1" || name == "h2" || name == "h3")` is usually faster then `Contains` for simple small checks